Calibration/EcalCalibAlgos/src/miscalibExample.cc

```diff
@@ -31,32 +29,6 @@
 #include <stdexcept>
 #include <vector>
```

Comment:
I see that this still uses `getByLabel()` to get collections from the event. This is deprecated as well and should be changed to the `getByToken()` method.
There is also a new `TH1F` created that is never deleted.

Reply:
> There is also a new TH1F created that is never deleted.

Now deleted 

> I see that this still uses getByLabel() to get collections from the event. This is deprecated as well and should be changed to the getByToken() method.

Ok - for this next commit I haven't updated it yet but will try to.